eventsourcing/pom.xml

```diff
@@ -39,6 +39,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.2</version>
```

Comment:
I think the dependency should be marked as optional, to prevent inclusion of a jar that may potentially conflict with classes already present in the JRE.

Reply:
marked